package.json

```diff
@@ -48,8 +48,8 @@
     "mkdirp": "^0.5.1",
     "mocha": "^5.2.0",
     "mustache": "^3.0.0",
-    "phantomjs-prebuilt": "^2.1.16",
     "prettysize": "^1.1.0",
+    "puppeteer": "^2.0.0",
```

Comment:
If phantom/puppeter is used only to find SVG dimensions when it missing, it could be moved to `optionalDependencies`

Reply:
Fully agree, viewbox is a good practice and forcing 200MB downloads for everyone who follow good practices is bad. What about to throw a meaningful message when puppeteer is needed but not installed?